src/foam/core/console/FileUpload.js

```diff
@@ -0,0 +1,1179 @@
+/**
+ * @license
+ * Copyright 2025 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.core.console',
+  name: 'DAOSuggestion',
+  
+  properties: [
+    {
+      class: 'String',
+      name: 'ID'
+    },
+    {
+      class: 'String', 
+      name: 'name'
+    },
+    {
+      class: 'String',
+      name: 'displayName'
+    },
+    {
+      class: 'Int',
+      name: 'score'
+    }
+  ],
+  
+  methods: [
+    function toSummary() {
+      return this.displayName;
+    }
+  ]
+});
+
+foam.CLASS({
+  package: 'foam.core.console',
+  name: 'FileUpload',
+  extends: 'foam.u2.Controller',
+
+  requires: [
+    'foam.dao.MDAO',
+    'foam.core.console.ColumnParser',
+    'foam.core.console.Mapping',
+    'foam.core.console.MappingsView',
+    'foam.core.console.PropertyChoiceView',
+    'foam.core.console.UploadService',
+    'foam.core.console.DAOHolder',
+    'foam.core.boot.CSpec',
+    'foam.core.fs.fileDropZone.FileDropZone',
+    'foam.core.fs.File',
+    'foam.u2.ActionView',
+    'foam.u2.view.RichChoiceView',
+    'foam.core.console.DAOSuggestion',
+    'foam.core.controller.Fonts',
```

Comment:
You dont need to require this class to use it

Reply:
got you, thank you